modules/oe_webtools_page_feedback/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenEuropa Webtools Page Feedback
+
+Provides a widget that integrates with the Webtools Default Feedback Form allowing the users to report on the page
+usefulness and potentially provide suggestions to improve the content.
+
+### How it works
+
+This module outputs the Webtools Default Feedback Form on all the content types when enabled.
+More information on this service can be found [here](https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Default+Feedback+Form).
+
+
+### How to use
+
+#### Required configuration
+
+In order to have the Webtools Default Feedback Form widget available on your site the OpenEuropa Webtools Page Feedback
+module needs a default configuration in the form of two variables:
+
+* Enabled: Whether the Page Feedback Form is enabled or not.
+* Form ID: The Webtools ID of the corresponding Feedback Form where submissions will be recorded.
+
+This configuration can be provided using the configuration form (`/admin/config/system/oe_webtools_page_feedback`) by
+users that were granted with the permission `administer webtools page feedback form` or by providing details in your
+site's settings.php file:
+
+```
+$config['oe_webtools_page_feedback.settings']['enabled'] = TRUE;
+$config['oe_webtools_page_feedback.settings']['formID'] = '1234';
```

Comment:
Usually, we don't use camel cases in elements of the configuration.
Using `form_id` probably would be more consistent.

Reply:
But in general, it is fine for me to have this variant as well.

Reply:
Lets make this change, I agree with Sergei in this case.